### PR TITLE
webex: Add version 41.12.0.20899

### LINKED ISSUE #5167: [Request] Add Webex
Some other video chat apps (Zoom, Discord) are available so it would be nice if Webex was in the bucket also: <https://www.webex.com/downloads.html>

### LINKED ISSUE #7117: (Request) add cisco-webex
PackageIdentifier:cisco-webex
PackageVersion: 39.3
PackageLocale: en-US
Publisher: cisco
PackageName: cisco-webex
License: proprietary
ShortDescription: meeting app
Installers:

Architecture: x64
InstallerType: msi
InstallerUrl: https://binaries.webex.com/WebexTeamsDesktop-Windows-Gold/Webex.msi
ManifestType: singleton
ManifestVersion: 1.0.0

### PR DESCRIPTION
**Pull request by MathewBlock**
Closes #5167

* Added auto updating manifest for Cisco Webex

### CODE REVIEW

File: bucket/webex.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "version": "41.12.0.20899",
+    "description": "Video and audio conferencing",
+    "homepage": "https://www.webex.com/",
+    "license": {
+        "identifier": "Proprietary"
+    },
```
Comment (rashil2000) [inline bucket/webex.json:7]: Add a URL field too, to their License/Privacy Policy/EULA.

File: bucket/webex.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "extract_dir": "Cisco Spark",
+    "shortcuts": [
+        [
+            "CiscoCollabHost.exe",
+            "Webex"
+        ]
+    ],
+    "checkver": {
+        "url": "https://help.webex.com/article/mqkve8",
+        "regex": "Version:\\s*((?:\\d+\\.)+(?:\\d+))"
```
Comment (rashil2000) [inline bucket/webex.json:27]:
```suggestion
        "regex": "Version:\\s([\\d.]+)"
```

Can be simplified

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/webex.json (hunk 1 above)
Comment (MathewBlock) [reply bucket/webex.json]: Initially I couldn't find one for the application online, but I managed to dig a little deeper and get the EULA.

File: bucket/webex.json (hunk 2 above)
Comment (MathewBlock) [reply bucket/webex.json]: Thanks, I'll make that change.

**Comment by Luckz** [comment]
It's dead, Jim: https://github.com/ScoopInstaller/Extras/issues/7994

**Pull request merged**